### ISSUE TITLE
File tree allows multiple starred values

### ISSUE THREAD

**Issue by abrookbanks** [cubecart/v6#1285]
It should allow one max.

**Comment by bhsmither** [comment]
I'm not seeing this as fixed.

My observation prior to adding this code is that:

Two files are listed on the Digital Options tab. The first one is checked (checked box image) as this choice is current.

I can check the second file (click the unchecked box image) and then both files are checked. As long as I do not disturb the first checkbox, I can check and uncheck the second box repeatedly.

This sequence did not change after adding the new code and making sure the browser has loaded the javascript (which it does anyway via `plugins.php?hash`).

What I see is that:
```
<input id="download_3" class="unique toggle" type="hidden" rel="3" name="download[3]" value="1">
```
is not part of the DOM until:
```
<img class="imgtoggle checkbox unique" src="admin/skins/default/images/0.png" rel="#download_3">
```
is actually clicked on. Then `<input>` shows up and `<img>` changes to:
```
<img class="checkbox unique toggle" src="admin/skins/default/images/1.png" rel="#download_3">
```
(as it is suppose to).

Was this fix simply to replace `star.png` with `1.png`?

**Comment by abrookbanks** [comment]
Definitely working on my 6.1 dev store.

You could install it from Master branch.

**Comment by abrookbanks** [comment]
I'll check again on Monday to be sure.

**Comment by abrookbanks** [comment]
I found an issue with digital toggles resetting image toggles. 

As of writing this seems to work well. See movie:
http://static.cubecart.com/issue-1285.mp4

**Comment by bhsmither** [comment]
Sorry about my comment above. I was working *solely* with the Digital tab.
On stock CC6012, even with fix b0e0849 applied:
0. Have several digital product files (PDFs, MP3s, etc) uploaded and ready to be assigned to products.
1. Have a digital product with a digital product file already assigned to it.
2. Fetch the Edit Product page for that product. (Gets a fresh DOM.)
3. View the Digital tab. Verify there is (an image of) a checked checkbox for the assigned file.
4. Click on any other (image of) an unchecked checkbox.
5. Note that the already checked checkbox does not uncheck until it eventually gets clicked on.

Which is slightly confusing, but does not affect what gets POSTed.

I have not looked at fix 2e3c7d3 as yet.
